consensus/propagation/catchup.go

```diff
@@ -100,7 +100,7 @@ func (blockProp *Reactor) AddCommitment(height int64, round int32, psh *types.Pa
 		blockProp.proposals[height] = make(map[int32]*proposalData)
 	}
 
-	combinedSet := proptypes.NewCombinedPartSetFromOriginal(types.NewPartSetFromHeader(*psh), true)
+	combinedSet := proptypes.NewCombinedPartSetFromOriginal(types.NewPartSetFromHeader(*psh, types.BlockPartSizeBytes), true)
```

Comment:
not blocking

do we need to pass this value or can we always just use this constant everywhere

Reply:
the only reason to pass the value is tests. in some places we're using different values in tests.